.github/workflows/test.yml

```diff
@@ -28,6 +28,8 @@ jobs:
     - run: yarn
     - run: yarn lint
     - run: yarn build
+    - if: ${{ startsWith(github.ref, 'refs/tags/') }}
```

Comment:
Given the high number of files downloaded by the script, its execution may take some time. 

Since the icons in the package are not used internally (i.e., the execution of `build-icons` is not needed to run `build` or `test` successfully), to avoid clogging CI runs I set the script to run only on tags

Reply:
This means that an error in the script can only be detected after a tag has been pushed to origin 🤔 

Can't we use caching to prevent building every time?

Reply:
Good point, we can use cache for sure. I'll look into that

Reply:
we decided to open an issue on this